.circleci/config.yml

```diff
@@ -7,40 +7,97 @@ templates:
     working_directory: /go/src/github.com/DataDog/pupernetes
 
 jobs:
-  build:
+  make:
     <<: *job_template
     steps:
       - checkout
       - run:
-          name: build
+          name: make
           command: make
 
   tests:
     <<: *job_template
     steps:
       - checkout
       - run:
-          name: run tests
+          name: tests
           command: make check
 
-  verify:
+  gofmt:
     <<: *job_template
     steps:
       - checkout
       - run:
           name: verify-gofmt
           command: make verify-gofmt
+
+  docs:
+    <<: *job_template
+    steps:
+      - checkout
       - run:
           name: verify-docs
           command: make verify-docs
+
+  license:
+    <<: *job_template
+    steps:
+      - checkout
       - run:
           name: verify-license
           command: make verify-license
 
+  setup:
+    machine: true
+    working_directory: /home/circleci/.go_workspace/src/github.com/DataDog/pupernetes
+    environment:
+      CFLAGS: ""
+      CGO_ENABLED: "1"
```

Comment:
Why do you keep `CGO_ENABLED=1`?

Reply:
I set that because of a circle ci issue:
```make
CGO_ENABLED=0 GOOS=linux go build -i -o pupernetes cmd/main.go
go install net: open /usr/local/go/pkg/linux_amd64/net.a: permission denied
go install os/user: open /usr/local/go/pkg/linux_amd64/os/user.a: permission denied
make: *** [pupernetes] Error 1
Exited with code 2```